src/illuminator/schema/simulation.py

```diff
@@ -17,6 +17,7 @@
 # monitor and connections sections enforce a format such as 
 # <model>.<input/output/state>
 valid_model_item_format = r'^\w+\.\w+$'
+# valid_model_item_format = r'^([\w-]+\.?)+$' # This is kept here as an alternative. I believe this might be useful later on
```

Comment:
Alternative regex in case connections might require more complex naming conventions

Reply:
That's ok, but if possible we should avoid complex names because the end user will struggle to remember the names while declaring models in the yaml file